packages/shared/src/components/utilities.tsx

```diff
@@ -118,11 +119,14 @@ export const pageBorders =
   'laptop:border-r laptop:border-l border-theme-divider-tertiary';
 const pagePaddings = 'px-4 tablet:px-8';
 
+export const pageContainerClassNames = classNames(
+  styles.pageContainer,
```

Comment:
Just a question:
Do we still needs these or can we potentially move them to tailwind classes now?

Reply:
I just checked they are pretty simple, we can probably convert them, though in a separate PR as it affects a lot of things and requires rigorous manual testing.